cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -27,27 +27,28 @@
 include(FindPackageHandleStandardArgs)
 include(SelectLibraryConfigurations)
 
-find_path(ZFP_INCLUDE_DIR
+find_path(ZFP_INCLUDE_DIRS
```

Comment:
DIR = cached & only one directory. This is the variable you will set.
DIRS = might be more than one directory and not cached. This is the variable you want to use.
In this way you are caching the wrong variable